xarray/tests/test_backends.py

```diff
@@ -753,6 +753,24 @@ def multiple_indexing(indexers):
         ]
         multiple_indexing(indexers)
 
+    def test_outer_indexing_reversed(self) -> None:
+        # regression test for GH6560
+        ds = xr.Dataset(
```

Comment:
This is the test of the issue, but it seems a bit excessive for a unit test to write so much data.
Can we construct a smaller dataset?